web_widget_json_graph/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Web Widget JSON Graph",
+    "version": "15.0.1.0.1",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "category": "Hidden/Dependency",
+    "website": "https://github.com/OCA/web",
+    "maintainers": ["frahikLV"],
```

Comment:
Don't remove maintainers. Add yourself to existing ones if that's what you want.

Reply:
Maybe you can add them like:

```python
"maintainers": [
    "luisg123v",
    "frahikLV",
],
```

Reply:
Done